ui/src/features/pipelines/modal/new-pipeline.component.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    ActionButton,
+    Button,
+    ButtonGroup,
+    Content,
+    Dialog,
+    DialogTrigger,
+    Divider,
+    Flex,
+    Grid,
+    Heading,
+    Item,
+    Menu,
+    MenuTrigger,
+    repeat,
+    Tag,
+    Text,
+    View,
+} from '@geti/ui';
+import { Calendar, MoreMenu } from '@geti/ui/icons';
+
+import { $api } from '../../../api/client';
+import Background from './../../../assets/background.png';
+import { NewPipelineButton } from './new-pipeline-button.component';
+
+export const NewPipeline = () => {
+    const { data: pipelinesData } = $api.useQuery('get', '/api/pipelines');
+
+    return (
+        <DialogTrigger type='fullscreenTakeover'>
+            <Button variant='secondary'>New Pipeline</Button>
+            {(close) => (
+                <Dialog
+                    UNSAFE_style={{
+                        backgroundColor: 'var(--spectrum-gray-100)',
+                        backgroundImage: `url(${Background})`,
+                        backgroundBlendMode: 'luminosity',
+                        backgroundPosition: 'center',
+                        backgroundRepeat: 'no-repeat',
+                        backgroundSize: 'cover',
+                    }}
+                >
+                    <Content>
+                        <Heading
+                            level={1}
+                            marginBottom={'size-250'}
+                            UNSAFE_style={{
+                                textAlign: 'center',
+                                fontSize: 'var(--spectrum-global-dimension-font-size-700)',
+                            }}
+                        >
+                            Pipelines
+                        </Heading>
+
+                        <Text
+                            marginBottom={'size-1000'}
+                            UNSAFE_style={{
```

Comment:
with the amount of unsafe_style's here i think it's worth to add a new module.scss file for this